sshd-sftp/pom.xml

```diff
@@ -150,7 +150,6 @@
 		            <plugin>
 		                <groupId>org.apache.maven.plugins</groupId>
 		                <artifactId>maven-surefire-plugin</artifactId>
-                        <version>${surefire.plugin.version}</version>
```

Comment:
Why was it necessary to remove this ? If I remember correctly there are quite a few artifacts that need to use this version version.

Reply:
The maven-surefire-plugin version is pinned to ${surefire.plugin.version} in the parent <pluginManagement> of the parent pom. It is therefore unnecessary and in fact bad practice to define it again here.

Reply:
Got it...